CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ Changelog
 - fixed `EventManager` not waiting for platform events websocket connection during initialization
 - fixed local `RequestQueueClient` not respecting the `forefront` argument
 - fixed local `RequestQueueClient` not counting the `handledRequestCount` property
+- fixed local `MemoryStorage` not handling the difference between storage `id` and `name` correctly
```

Comment:
Right, I'll move it to a `Breaking changes` section. It shouldn't be too breaking, since almost noone is using the SDK just yet, but better to be explicit about it.